api/src/main/java/module-info.java

```diff
@@ -798,7 +798,11 @@
  * For example,</p>
  *
  * <pre>
- * Product[] findByNameLike(String pattern, PageRequest pageRequest, Order&lt;Product&gt; order);
+ * Product[] findByNameLikeAndPriceBetween(String pattern,
+ *                                         float minPrice,
+ *                                         float maxPrice,
+ *                                         PageRequest pageRequest,
+ *                                         Order&lt;Product&gt; order);
```

Comment:
FYI - Here is the example in the Javadoc that this is supposed to correspond to:

```
 * page1 = products.findByNameLikeAndPriceBetween(
 *                 namePattern, minPrice, maxPrice, page1Request,
 *                 Order.by(Sort.desc("price"), Sort.asc("id"));
```